script.subtitles.zimukux/tests/test_zimuku.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
 
     def get_agent(self, settings):
         return zmkagnt.Zimuku_Agent(
-            'http://zimuku.org', tmp_folder, Logger(),
```

Comment:
please keep the tests folder out of the submission too

Reply:
removed